python/mleap/sklearn/preprocessing/data.py

```diff
@@ -648,20 +670,19 @@ def serialize_to_bundle(self, transformer, path, model_name):
     def deserialize_from_bundle(self, transformer, node_path, node_name):
 
         attributes_map = {
-            'size': 'n_values_'
+            'size': 'categories_',
```

Comment:
We should make a note that models serialized with older versions of mleap will not be deserialized correctly

Reply:
Where would be the best place to do this? In the release notes?

Reply:
release notes has a breaking change section that we could use for this. i've added them in for the future release, 0.17.0 in this PR https://github.com/combust/mleap/pull/736.